Google.Api.Gax.Grpc.Tests/Rest/transcoder_tests.json

```diff
@@ -317,6 +317,22 @@
         "uri": "/abc/def?enumField=FIRST_VALUE&requiredInteger=0"
       }
     },
+    {
+      "name": "ComplexQuery_UnknownEnumValues",
+      "details": "Enum values are represented with their original names",
```

Comment:
```suggestion
      "details": "Enum values are represented with their numeric values",
```

Reply:
Whoops - fixed (with slightly different wording, but basically it's there)